.github/workflows/coverage.yml

```
@@ -73,6 +18,7 @@ jobs:
       - uses: actions/checkout@v4
         with: 
           persist-credentials: false
+          contents: read
```

actionlint (1.7.12)

[error] input "contents" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)